citylearn/building.py

```diff
@@ -1460,15 +1561,23 @@ def estimate_action_space(self) -> spaces.Box:
                 low_limit.append(0.0)
                 high_limit.append(1.0)
             
+            elif "ev_storage" in key:
```

Comment:
For consistency, use single quote for strings. Check everywhere also in other modules for this.